### DISCUSSION TITLE
Details on Eltako Telegramms // how can i help adding them

### DISCUSSION THREAD

**Discussion by sipodd** [mak-gitdev/HA_enoceanmqtt#59] [Ideas]
Hi, 
i really like your HA Addon!
Still got some difficulties, but i guess its more me then your Addon.

I'd like to share a part of the Eltako Documentation with you, it contains all their messages and paylode at the end of the document.
[https://www.eltako.com/fileadmin/downloads/en/_main_catalogue/Gesamt-Katalog_ChT_gb_highRes.pdf](https://www.eltako.com/fileadmin/downloads/en/_main_catalogue/Gesamt-Katalog_ChT_gb_highRes.pdf)

I'm not a programmer, but if there is a way i can help by bringing such information into another format/file, i'd be glad to help.
Just need an explaination what to do :-)

Cheers Simon

**Comment by mak-gitdev** [comment]
Hi @sipodd,

Thanks ;).

I know about this file.
The problem with Eltako devices is that some of them are available out-of-the-box. But for the others, it requires to update the code.
I am working on it but at a slow pace as I have little time.

I won't hesitate to ask your help for testing. Thanks :+1:
